README.md

```diff
@@ -22,7 +22,7 @@ Our [GETTING_STARTED](GETTING_STARTED.md) and [CONTRIBUTING](CONTRIBUTING.md) gu
 
 ### Hacktoberfest :tada:
 
-Are you participating in [Hacktoberfest](https://hacktoberfest.digitalocean.com/)? Go check out our [Welcome New Developers project board][welcome-dev]! We have curated all issues we consider to be a good starting point for new members of the community and categorized them by skills and level of complexity. 
+Are you participating in [Hacktoberfest](https://hacktoberfest.digitalocean.com/)? Go check out our [Welcome New Developers project board](https://github.com/orgs/openfoodfoundation/projects/5)! We have curated all issues we consider to be a good starting point for new members of the community and categorized them by skills and level of complexity. 
```

Comment:
An alternative to updating the link here is to update the footnote link on line 55

Reply:
Thanks Sigmund - was unaware we could do footnotes on Markup, done now.